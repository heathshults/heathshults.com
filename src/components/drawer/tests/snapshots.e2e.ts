import { snapIt } from '../../../test';

const component = 'drawer';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('renders correctly with no props', '<hs-drawer>default</hs-drawer>');
    snap('renders as open', '<hs-drawer open>default</hs-drawer>');
    snap('renders dismissible overlay', '<hs-drawer open dismissible>default</hs-drawer>');
  });
});
