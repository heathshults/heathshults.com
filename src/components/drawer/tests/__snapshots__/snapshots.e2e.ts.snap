// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`drawer renders renders as open 1`] = `
"<hs-drawer open=\\"\\" class=\\"hydrated\\"><div aria-hidden=\\"true\\" class=\\"hs-overlay hs-overlay--fullpage hs-overlay--visible\\"></div><aside aria-expanded=\\"async isOpen() {
        return this._isOpen;
    }\\" class=\\"hs-drawer hs-drawer--bottom hs-drawer--visible\\">default</aside></hs-drawer>"
`;

exports[`drawer renders renders correctly with no props 1`] = `
"<hs-drawer class=\\"hydrated\\"><div aria-hidden=\\"true\\" class=\\"hs-overlay hs-overlay--fullpage\\"></div><aside aria-expanded=\\"async isOpen() {
        return this._isOpen;
    }\\" class=\\"hs-drawer hs-drawer--bottom\\">default</aside></hs-drawer>"
`;

exports[`drawer renders renders dismissible overlay 1`] = `
"<hs-drawer open=\\"\\" dismissible=\\"\\" class=\\"hydrated\\"><div aria-hidden=\\"true\\" class=\\"hs-overlay hs-overlay--fullpage hs-overlay--visible\\"></div><aside aria-expanded=\\"async isOpen() {
        return this._isOpen;
    }\\" class=\\"hs-drawer hs-drawer--bottom hs-drawer--visible\\">default</aside></hs-drawer>"
`;
